src/bare_etiss_processor/main.cpp

```diff
@@ -149,8 +148,12 @@ int main(int argc, const char *argv[])
 
     if (!quiet) std::cout << "=== Setting up plug-ins ===" << std::endl;
 
-    auto irq_handler = std::make_shared<etiss::InterruptHandler>(cpu->getInterruptVector(), cpu->getInterruptEnable(), cpu->getArch(), etiss::LEVEL_TRIGGERED, false);
-    cpu->addPlugin(irq_handler);
+    bool enable_irq_handler = etiss::cfg().get<bool>("vp.enable_irq_handler", true);
```

Comment:
Defaulting to `true` to make sure taht the original behavior does not change. However, this impacts simulation performance if the plugin in this required, hence at some point I would prefer to switch to `false`...